src/pages/blocks/index.tsx

```diff
@@ -20,7 +20,6 @@ export default function Blocks ({ blocks }: InferGetServerSidePropsType<typeof g
   return (
     <div className='container mx-auto px-4 pt-12 pb-20'>
       <Head title='Blocks' />
-
```

Comment:
remove this change to prevent upstream conflicts

Reply:
Done.